docs/plugins.md

```
@@ -0,0 +1,375 @@
+plugins/
+  example/
+    plugin.toml
+    kopuz-plugin-example
+```
+
+`KOPUZ_PLUGIN_PATH` adds further roots, separated the way `PATH` is, scanned
+after the config directory. Packaged builds want this: under Nix, Flatpak or any
+other store-based install the plugin sits on a read-only path with nowhere to
+copy it from, so the package sets the variable instead.
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)